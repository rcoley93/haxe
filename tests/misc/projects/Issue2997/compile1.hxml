--display Main.hx@138@position