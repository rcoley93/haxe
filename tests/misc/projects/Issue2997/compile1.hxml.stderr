<list>
<pos>$$normPath(::cwd::/Main.hx):2: characters 4-14</pos>
</list>